docs/content/010-getting-started/03-tutorial/06-chapter-5-persisting-data-via-prisma.mdx

```diff
@@ -223,15 +222,16 @@ export function createContext(): Context {
 
 ```ts
 // api/context.ts
-import { PrismaClient } from '@prisma/client'
+import { db } from "./db";
+import { PrismaClient } from "@prisma/client"
 
 export interface Context {
   db: PrismaClient
 }
 
 export function createContext(): Context {
   return {
-    db: new PrismaClient(),
+    db,
```

Comment:
Can you update this to match https://github.com/graphql-nexus/tutorial/blob/master/api/context.ts

I don't think we need a db modular just to instantiate a Prisma client :)